presentation.js

```diff
@@ -15,10 +15,34 @@ define(function(require, exports, module) {
 
         var plugin = new Plugin("CS50", main.consumes);
 
-        var presenting = false;
+        var presenting = null;
```

Comment:
Why `null`? Sounds like a `Boolean`, no?

Reply:
Wanna differentiate between the case where `presenting` is initialized or not.

Reply:
How's this help, though? You can still check for `undefined`. `null` feels like the wrong semantics, since this ultimately contains a `Boolean`, not an `Object` more generally?

Reply:
Changed to `undefined`.

Reply:
PS, it helps in case the listener was registered after `presenting` was initialized. We need to call the listener immediately in this case to have the target plugin handle current presentation mode state accordingly.